### DISCUSSION TITLE
Using DDrawcompat specific options with dxwrapper

### DISCUSSION THREAD

**Discussion by TerrorizedGakon** [elishacloud/dxwrapper#251] [Q&A] [answered]
Hello,

Does dxwrapper support options of ddrawcompat, like FpsLimiter? Or reading the config of DDrawCompat?

I can't seem to get it work. I have a game which requires both ddraw( GDI from the logs? I'm not a programmer, so I don't really know) and d3d8 hooking. I can have everything working fine when I use narzouls ddraw and also your d3d8 stub with your dxwrapper, but I'm curious if it's all possible with just dxwrapper injecting narzouls DDrawCompat.

**Comment by elishacloud** [comment, marked as answer]
No.  The DDrawCompat I have included in dxwrapper is only up until v3.2, which does not have any settings.  Settings were first added in DDrawCompat v4.0, which is not included in dxwrapper.

**Reply by TerrorizedGakon** [reply]
Thank you for your explanation :)
